.repo-metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "bigtable-hbase",
```

Comment:
what is this name used for? Just pattern matching this looks different from the name in the native client:

In the native client:
repo name: java-bigtable
metadata name: google-cloud-bigtable

Here:
repo name: java-bigtable-hbase
metadata name: bigtable-hbase

Reply:
This was only used for generating the bug report template. https://github.com/googleapis/java-bigtable-hbase/pull/2412/commits/30c6aa041f70e56c5b9412aef03e4d0f7880b76b#diff-74c2649c999338ed7a2833d79d4b25c3R20. I've cleaned that up a little and added it to exclude from the generation.